src/app/pages/questions/components/question/info-screen/info-screen.component.ts

```diff
@@ -53,12 +53,12 @@ export class InfoScreenComponent implements OnInit {
 
   scrollDown() {
     const dimensions = this.content.getContentDimensions()
-    const position = dimensions.scrollTop + dimensions.contentHeight
+    const position = dimensions.scrollTop + dimensions.contentHeight / 2
     this.content.scrollTo(0, position, 1000)
   }
 
   onScroll(event) {
-    if (event.scrollTop >= event.scrollHeight - event.contentHeight) {
+    if (event.scrollTop >= (event.scrollHeight - event.contentHeight) * 0.8) {
```

Comment:
how are these values calculated ? like why 0.8?

Reply:
I thought maybe if they have scrolled through 80% of the info screen then the 'Next' button would already activate instead of having to scroll through 100% since most of the bottom parts are spaces.

Reply:
ok